tests/perf_test_harness.rs

```diff
@@ -0,0 +1,18 @@
+#![allow(clippy::unwrap_used, clippy::expect_used)]
+use anyhow::Result;
+use fixtures::ctx::E2ECtx;
+
+mod fixtures;
+
+#[test_log::test(tokio::test)]
+#[ignore = "This is intended to easily start a few nodes locally for devs"]
+async fn setup_for_perf_test() -> Result<()> {
```

Comment:
is it specific for perf test only ? It can serve general purpose no ? Would name that setup_4_nodes or smth like that